yaml/roads.yaml

```diff
@@ -78,6 +78,14 @@ global:
       sidewalk: {col: sidewalk}
       sidewalk_left: {col: 'sidewalk:left'}
       sidewalk_right: {col: 'sidewalk:right'}
+      cutting:
+        case:
+          - when: {cutting: ['yes', 'right', 'left', 'both']}
+            then: true
+      embankment:
+        case:
+          - when: {embankment: ['yes', 'right', 'left', 'two_sided', 'both']}
```

Comment:
Can this work more like bike lanes – where it's yes, right, left and if both then it'd just be yes? (Please verify, this is from memory.)